app/assets/stylesheets/components/_calendars.scss

```diff
@@ -24,13 +24,14 @@ $calendar-bookable-slot-background: $color-madang;
 
 .company-calendar {
   .fc-resource-cell {
-    height: 150px;
+    height: 120px;
```

Comment:
Might be good to just make this `120px` a variable `$guider-row-height` or something?